src/js/components/Hello.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+
+class Hello extends Component {
+
+  	constructor() {
+    	super();
+  	}
+
+  	render() {
+    	return (
+      		<h1 id="hello-react-para">hello react!</h1>
+    	);
+  	}
+
+}
+
+export default Hello;
+
+const hello = document.getElementById("hello-react");
```

Comment:
why the need for 'hello ?' below? you should be able to ensure this is rendered when the DOM is ready